gulpfile.js

```diff
@@ -174,7 +174,7 @@ async function updateBuildNumber(args) {
             versionParts.length > 2 ? versionParts[2].replace(/(\d+)/, args.buildNumber) : args.buildNumber;
         const newVersion =
             versionParts.length > 1
-                ? `${versionParts[0]}.${versionParts[1]}.99${buildNumberPortion.substring(0, buildNumberPortion.length - 2)}`
+                ? `${versionParts[0]}.${versionParts[1]}.999${buildNumberPortion.substring(0, buildNumberPortion.length - 3)}`
```

Comment:
This is to force the higher version number? It doesn't look like a guarantee here to me it's unlikely, but we might have already generated a 999 build version, right?